modules/adapter/templates/st2_gunicorn_init/systemd.service.erb

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=StackStorm <%= @_subsystem %> gunicorn script.
+After=network.target
+
+[Service]
+Type=simple
+Environment="PYTHONPATH=<%= @_python_pack %>/<%= @_subsystem %>/<%= @_subsystem %>"
+ExecStart=gunicorn_pecan <%= @_python_pack %>/<%= @_subsystem %>/gunicorn_config.py \
```

Comment:
It's a bit odd you use relative binary paths with upstart. So I just copy-pasted, maybe this will also work with systemd.